pages/article/ArticleHandler.inc.php

```diff
@@ -182,9 +270,10 @@ function view($args, $request) {
 			$templateMgr->assign('galley', $galley);
 			$templateMgr->assign('section', $section);
 			$templateMgr->assign('journal', $journal);
-			$templateMgr->assign('fileId', $fileId);
 			$templateMgr->assign('defineTermsContextId', isset($defineTermsContextId)?$defineTermsContextId:null);
-
+			
```

Comment:
What about keywords in the line 265 -- will there be the right version of keywords?